Makefile

```diff
@@ -101,6 +119,12 @@ fmt: ## Run go fmt against code.
 vet: ## Run go vet against code.
 	go vet -mod=vendor ./...
 
+.PHONY: iamctl-gen
+iamctl-gen: iamctl-build
+	$(IAMCTL_BINARY) genpolicy -i $(IAMCTL_ASSETS_DIR)/iam-policy.json -o $(IAMCTL_OUTPUT_DIR)/$(IAMCTL_OUTPUT_FILE) -p $(IAMCTL_GO_PACKAGE)
+	go fmt -mod=vendor ./...
```

Comment:
Why is is necessary to run `fmt` and `vet` on all the packages after generating the policy.

Reply:
done